modules/preview-react/table/lib/TableBody.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Grid, GridProps} from '@workday/canvas-kit-react/layout';
+import {createComponent} from '@workday/canvas-kit-react/common';
+
+/**
+ * `Table.Body` renders a [tbody](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody) element.
+ *
+ * @example
+ * ```tsx
+import {Table} from '@workday/canvas-kit-preview-react/table';
+
+export default function App() {
+  return (
+    <Table>
+      <Table.Body>
+        <Table.Row>
+          <Table.Header>Table Header</Table.Header>
+          <Table.Cell>Table Cell</Table.Cell>
+        </Table.Row>
+      </Table.Body>
+    </Table>
+  );
+}
+```
+ */
```

Comment:
I'm still learning here, but if we have the same documentation at the `Tabel` level do we still need it here?

Reply:
I don't think we do. It's redundant and I will update it.